scripts/system/create/entityProperties/html/js/entityProperties.js

```diff
@@ -133,11 +141,21 @@ const GROUPS = [
                 type: "color",
                 propertyID: "color",
             },
+            {
+                label: "Alpha",
+                type: "number-draggable",
+                min: 0,
+                max: 1,
+                step: 0.01,
+                decimals: 2,
+                propertyID: "shapeAlpha",
+                propertyName: "alpha",
+            },            
```

Comment:
This is being added in https://github.com/kasenvr/project-athena/pull/215?
Though the UI control for propery appears to work OK in this PR, so perhaps could be left in.

Reply:
Same reason.
Should work except for the tooltip.
it will come with the PR 215